### ISSUE TITLE
resilver_finish-notify.sh exits with status 9 due to no ZEVENT_POOL

### ISSUE THREAD

**Issue by klkblake** [zfsonlinux/zfs#6026]
### System information
Type                                | Version/Name
  ---                                  |     --- 
Distribution Name       | Funtoo Linux
Distribution Version    | N/A (rolling release)
Linux Kernel                 | 4.9.20-gentoo
Architecture                 | amd64
ZFS Version                  | 0.7.0-rc3
SPL Version                  | 0.7.0-rc3

### Describe the problem you're observing
resilver_finish-notify.sh exits with status 9 without sending the notification. Examining /tmp/zed.debug.log reveals that no ZEVENT_POOL variable is provided. The script appears to be checking for the presence of that variable and exiting immediately when it is not found.

### Describe how to reproduce the problem
Set up a notification method in zed.rc, offline and online a disk to trigger an incremental resilver, check /var/log/messages to see that it exited with exit code 9 without actually triggering the notification

### Include any warning/errors/backtraces from the system logs
The contents of /tmp/zed.debug.log:
```
IFS= 	
PATH=/usr/bin:/bin:/usr/sbin:/sbin
PWD=/
SHLVL=1
ZDB=/sbin/zdb
ZED=/sbin/zed
ZED_PID=12847
ZED_ZEDLET_DIR=/etc/zfs/zed.d
ZEVENT_CLASS=sysevent.fs.zfs.resilver_finish
ZEVENT_EID=16
ZEVENT_POOL_CONTEXT=0
ZEVENT_POOL_GUID=0xDA3AAD673F3CF209
ZEVENT_SUBCLASS=resilver_finish
ZEVENT_TIME=1492245197 82931139
ZEVENT_TIME_NSECS=82931139
ZEVENT_TIME_SECS=1492245197
ZEVENT_TIME_STRING=2017-04-15 18:03:17+0930
ZEVENT_VERSION=0
ZFS=/sbin/zfs
ZFS_ALIAS=zfs-0.7.0-rc3
ZFS_RELEASE=rc3
ZFS_VERSION=0.7.0
ZINJECT=/sbin/zinject
ZPOOL=/sbin/zpool
_=/usr/bin/printenv
```

**Comment by SenH** [comment]
I'm experiencing the same exit status preventing sending notifications upon completing a scrub (ZFS 0.7.0-rc3_161_g2035575fd).

```
zed: eid=5 class=scrub_finish pool_guid=0x909A3B2CC8027003  
zed[15506]: Finished "all-syslog.sh" eid=5 pid=31111 exit=0
zed[15506]: Invoking "scrub_finish-notify.sh" eid=5 pid=31149
zed[15506]: Finished "scrub_finish-notify.sh" eid=5 pid=31149 exit=9
```

**Comment by behlendorf** [comment]
Thanks for the bug report.  This is resolved in zfs-0.7.0-rc3-168-gbcdb96a and newer version where `ZEVENT_POOL` is unconditionally added to all events.
